composer.json

```diff
@@ -11,7 +11,7 @@
         {"name": "Michael Simonson", "email": "contact@mikesimonson.com" }
     ],
     "require": {
-        "php": ">=5.4.0",
+        "php": "^5.5|^7.0",
```

Comment:
`^5.5` === `>= 5.5.0 && < 6`
`~5.5` === `>= 5.5.0 && < 6`

The difference only is visible when specifying a patch version:
`^5.5.1` === `>= 5.5.1 && < 6`
`~5.5.1` === `>= 5.5.1 && < 5.6`